Assets/Editor/TestShopify.cs

```diff
@@ -29,6 +29,14 @@ public class TestShopify {
             Assert.AreEqual("domain.com", ShopifyBuy.Client().Domain);
             Assert.AreEqual("AccessToken", ShopifyBuy.Client().AccessToken);
         }
+        public void TestUpdateLocale() {
+            ShopifyBuy.Init("AccessToken", "domain.com");
```

Comment:
Perform some assertions on the updated `locale` before and after change. Not sure how you can access the `locale`. Maybe via the query loader?  

```suggestion
            string initialLocale = "en";
            string updatedLocale = "fr";
            ShopifyBuy.Init("AccessToken", "domain.com");
            Assert.AreEqual(initialLocale, ShopifyBuy.Client().Loader.Locale)
            
            ShopifyBuy.Client("domain.com").UpdateLocale("fr");

            Assert.IsNotNull(ShopifyBuy.Client());
            Assert.AreEqual("domain.com", ShopifyBuy.Client().Domain);
            Assert.AreEqual("AccessToken", ShopifyBuy.Client().AccessToken);
            Assert.AreEqual(updatedLocale, ShopifyBuy.Client().Loader.Locale)
```